src/detext/run_detext.py

```diff
@@ -114,6 +115,7 @@ def add_arguments(parser):
     # Misc
     parser.add_argument("--random_seed", type=int, default=1234, help="Random seed (>0, set a specific seed).")
     parser.add_argument("--steps_per_stats", type=int, default=100, help="training steps to print statistics.")
+    parser.add_argument("--total_eval_steps", type=int, default=None, help="total number of evaluation steps")
```

Comment:
Let's not name it as `total_eval_steps` since the meaning of `steps` here is different than the steps used in `steps_per_stats`, `num_train_steps`, etc..

After reading your implementation, I found that this param means "the number/round of evaluation" and it does not relate to "total".

Suggest
1. changing the naming and the param description
2. add into the description that this param will overwrite steps_per_eval

Reply:
Thanks for your comment, what about call it **eval_round** ?

Reply:
or **num_eval_round**?

Reply:
Both names look okay to me